examples/offline_inference_molmo.py

```diff
@@ -0,0 +1,129 @@
+import argparse
+import numpy as np
+import requests
+from io import BytesIO
+import base64
+from PIL import Image, ImageFile, ImageOps
+import torch
+from typing import Optional
+
+from vllm import LLM
+from vllm.sampling_params import SamplingParams
+
+
+ImageFile.LOAD_TRUNCATED_IMAGES = True
+
+
+def download_image_to_numpy(url):
+    # Send a GET request to the URL
+    response = requests.get(url)
+    
+    # Check if the request was successful
+    if response.status_code == 200:
+        # Open the image from the response content
+        image = Image.open(BytesIO(response.content)).convert("RGB")
+
+        image = ImageOps.exif_transpose(image)
+        
+        # Convert the image to a NumPy array
+        image_array = np.array(image).astype(np.uint8)
```

Comment:
I was able to run without these lines at all as the preprocessor in HuggingFace performs the same conversion, it might be a good suggestion to just remove them?

Reply:
That's correct, it's applied in the image processor and we'll remove that part.